lnrpc/signrpc/signer.proto

```diff
@@ -380,6 +384,10 @@ message VerifyMessageReq {
     Specifies if the signature is a Schnorr signature.
     */
     bool is_schnorr_sig = 4;
+
+    // An optional tag that can be provided when taking a tagged hash of a
+    // message. This option can only be used when is_schnorr_sig is true.
```

Comment:
miiiiicro nit: see how other multi line comments are done here usint `/* ... */` instead of `//`